<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reload">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload Data</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="total100">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AED100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total by batch</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="total150">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AED150</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="total200">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AED200</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="given100">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Already Given</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="given200">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="given150">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Voucher configuration</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="bExport">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="exportDateTo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bErase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Erase Data</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="eraseDateFrom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="eraseDateTo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="exportDateFrom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database configuration</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
